plugins/sparse_matrix_dataset.cc

```diff
@@ -482,6 +482,7 @@ struct SparseMatrixDataset::Itl
             return { hash, 24 };
         }
         default:
+            cerr << "WIERD TYPE: " << val.cellType() << endl;
```

Comment:
Let's make this error message easier for a user to action, by changing the ExcAssert(false) to an throw HttpReturnException(...)